context/app/static/js/components/home-revision/AssayTypeBarChart/AssayTypeBarChart.jsx

```diff
@@ -90,62 +94,110 @@ function AssayTypeBarChart({ parentWidth, parentHeight }) {
   docCountScale.rangeRound([0, xMax]);
   dataTypeScale.rangeRound([yMax, 0]);
 
+  const { tooltipData, tooltipLeft, tooltipTop, tooltipOpen, showTooltip, hideTooltip } = useTooltip();
+
+  const { containerRef, TooltipInPortal } = useTooltipInPortal({
```

Comment:
Can you clarify the name? This  whole thing is the "portal", so do you mean something more precise?

Reply:
This comes directly from the visx `useTooltipInPortal` hook (their name not ours) and doesn't refer to our portal, see https://reactjs.org/docs/portals.html .